tests/service/ProjectAnalysisTests.fs

```diff
@@ -3070,7 +3070,7 @@ let ``Test Project21 all symbols`` () =
             ("unit", "unit", "file1", ((12, 43), (12, 47)), ["type"], ["abbrev"]);
             ("val raise", "raise", "file1", ((13, 18), (13, 23)), [], ["val"]);
             ("System", "System", "file1", ((13, 25), (13, 31)), [], ["namespace"]);
-            ("member .ctor", "``.ctor``", "file1", ((13, 25), (13, 55)), [], ["member"]);
+            ("member .ctor", "NotImplementedException", "file1", ((13, 25), (13, 55)), [], ["member"]);
```

Comment:
Can we change this behavior? Seems more useful to me like this, but maybe I just don't know the use cases...

Reply:
Why should `su.Symbol.DisplayName` be a NotImplementedException here?

Reply:
Because that's what's actually displayed. It's a constructor for `NotImplementedException`.

Reply:
Sounds like the root cause is somewhere down the drain...

Reply:
Root cause is described in the PR description. Question is if we should fix it in `DisplayNameCore` or keep that as is and just add the extra logic to finding references. 

Second option is safer so maybe I just do that and we can merge it without worrying...

Reply:
Well to me the first option seems a more correct way to go since this outcome is weird and the problem in DisplayNameCore might have some other weird consequences therefore.

That being said, I would be fine with having this as is and creating a ticket to fix the DisplayNameCore.

Reply:
Ok, let's merge this and see what happens...